kotlin/workflow-core/src/main/java/com/squareup/workflow/RenderContext.kt

```diff
@@ -50,21 +50,21 @@ import com.squareup.workflow.WorkflowAction.Updater
  *
  * See [renderChild].
  */
-interface RenderContext<StateT, in OutputT : Any> : Sink<WorkflowAction<StateT, OutputT>> {
-
-  @Deprecated("Use RenderContext.send.")
-  fun <EventT : Any> onEvent(
-    handler: (EventT) -> WorkflowAction<StateT, OutputT>
-  ): (EventT) -> Unit
+interface RenderContext<StateT, in OutputT : Any> {
 
   /**
-   * Accepts a single [WorkflowAction], and invokes that action by calling [WorkflowAction.apply]
+   * Accepts a single [WorkflowAction], invokes that action by calling [WorkflowAction.apply]
    * to update the current state, and optionally emits the returned output value if it is non-null.
    *
    * This method is defined by the [Sink] interface. Since [RenderContext] implements [Sink],
    * operations like [contraMap] are available.
    */
-  override fun send(value: WorkflowAction<StateT, OutputT>)
+  val actionSink: Sink<WorkflowAction<StateT, OutputT>>
+
+  @Deprecated("Use RenderContext.send.")
```

Comment:
Should this be `RenderContext.actionSink.send`? Also, possible to add a replacement strategy?

Reply:
Oops

Reply:
#841